src/arcaflow_plugin_sdk/test_schema.py

```diff
@@ -584,6 +590,30 @@ def setUp(self):
             },
             "a",
         )
+        self.scope_inlined_int = schema.ScopeType(
+            {
+                "a": schema.ObjectType(
+                    InlineInt,
+                    {
+                        discriminator_field_name: PropertyType(
+                            schema.StringType(),
+                        ),
```

Comment:
Line 599 deserves a comment:  why does the discriminator field have a `StringType` when the object which contains it declares it to be an `int` at line 57?  Ditto line 609.

Reply:
oops, fixed it!

Reply:
Oh, good.  But...why did it work prior to you fix??  Um...nevermind...don't want to know!  🙉